### DISCUSSION TITLE
How do I lower MT-32 volume?

### DISCUSSION THREAD

**Discussion by ShmulikKraus** [joncampbell123/dosbox-x#5002] [General]
Hi, I'm using DOSBox-X to play "The Legend of Kyrandia (Book One)" I have setup the game to use MT-32 for music and SBPro for sound. The problem is that the music is at maximum volume and the sound (speech, effects, etc.) is very low, almost inaudible. When I go to "Sound->Show sound mixer volumes" it shows SB volume at 10%. I tried to exit the game and increase the SB volume with "mixer sb 100", but when I start the game it just returned to 10%.

How do I turn down MT-32 volume, or, even better, turn up SB volume?

**Comment by ShmulikKraus** [comment]
Never mind, this also happens in vanilla DOSBox so I'm guessing I don't even have MT-32 working correctly and instead the game uses General MIDI. 
If I understand correctly, there is no way to decrease MIDI volume without an external MIDI synth program because DOSBox just sends the output to the OS's MIDI synth (in my case it's "Microsoft GS Wavetable Synth").

I still don't understand why DOSBox (even vanilla) lowers SB volume when MIDI is used, but a workaround I found is simply increasing MASTER volume since the MASTER volume doesn't affect MIDI thus equalizing digital sound and MIDI.

Topic closed.

**Comment by aaronfranke** [comment]
Not closed - we still need way to lower MIDI volume. I can't hear my friends on Discord when DOSBox is playing music.
